src/views/triage/confirm.html

```diff
@@ -0,0 +1,30 @@
+<main id="content" tabindex="-1" role="main">
+
+  {{> common/betaBanner }}
+
+  {{> common/backLink }}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {{> common/errorSummary }}
+
+      {{> common/pageHeading pageHeading }}
+
+      {{#each triageData.selectedActivities.items}}
+        <p>{{this.text}}</p>
+      {{/each}}
+
+      {{#each triageData.selectedOptionalAssessments.items}}
+        <p>{{this.text}}</p>
+      {{/each}}
+
+      <form id="form" method="POST" action="{{formAction}}" novalidate="novalidate">
+        {{> common/csrfToken token=DefraCsrfToken}}
+
+        <button id="submit-button" type="submit" class="button" name="confirmed" value="confirmed">Confirm</button>
```

Comment:
Just wondering why you are not using the partial for the submit button:
{{> common/submitButton text="Confirm"}}

I can see the following in the triage controller:

            if (data.selectedOptionalAssessments) {
              // POSTing confirmation
              const requestedConfirmed = request.payload['confirmed']
              if (requestedConfirmed === 'confirmed') {
                // TODO - Save the application
                data.confirmed = true
              }
            } else {

But wouldn't the following do:

            if (data.selectedOptionalAssessments) {
              // POSTing confirmation
              // TODO - Save the application
              data.confirmed = true
            } else {

This is because posting here is a confirmation I would have thought.

Reply:
It's not essential, it was just to make the whole thing consistent as a "confirmed" parameter is also passed through the URL path and this way it just makes sure that it's named the same way everywhere.

But as you say, it doesn't have to be like that and this is just a holding page anyway until we've got the cost calculation in.